### ISSUE TITLE
Not work on windows.

### ISSUE THREAD

**Issue by xpol** [AtomLinter/linter-eslint#168]
I insert a log after [linter-eslint.js#L163](https://github.com/AtomLinter/linter-eslint/blob/master/lib/linter-eslint.js#L163) 

```javascript
    // Exec the CLI with correct params
    const output = await this.exec(eslint, params, {cwd})
   console.log("output:%s", output);
```

The log never get printed on windows.

**Comment by xpol** [comment]
btw, should it be `{cwd}` or `{cwd: cwd}` ?

**Comment by preco21** [comment]
Same issue

linter version: 1.2.3
linter-eslint version: 2.4.3

**Comment by Japskua** [comment]
Same problem :+1:

**Comment by jscissr** [comment]
@xpol `{cwd}` is just short for `{cwd: cwd}` in ES6. See [here](http://babeljs.io/repl/#?experimental=false&evaluate=false&loose=false&spec=false&code=exec(eslint%2C%20params%2C%20%7Bcwd%7D)).

**Comment by joemcbride** [comment]
Also not working for me.

linter version: 1.2.3
linter-eslint: 2.4.3

I also tried linter-eslint 2.4.2 and that did not appear to work either, though 2.4.1 does work.  So the commits for 2.4.2 could be the culprits.

**Comment by iam4x** [comment]
I've rollbacked linter to 1.x with the 3.0.0 version published. Please open a new issue if it's still happening.

Make sure to quit Atom and `$ apm uninstall linter-eslint && apm install linter-eslint`.

**Comment by xpol** [comment]
@iam4x Wait , please, I just worked hours this night and fixed this [here](https://github.com/xpol/linter-eslint/commit/0bd48b06d8c510971e929d7ad5bb2bdf840bbe8d). As you've rollbacked, I can't make a correct pull request. Would you help me please?

**Comment by iam4x** [comment]
I'm sorry about your work, I have rollbacked hours of work of myself too.

The first release of 2.0 was using atom-linter.exec and stdin, we had the
same performances issues. The fact is spawning a process is the culprit of
bad performances.

And using atom-linter.exec won't work when you start atom from launcher /
dock icon since node executable wont be in path.

I've tried lot of fixes and other ways to achieve linting and the best user
experience was with 1.x version.
